pkg/workloads/workloads.go

```diff
@@ -1,3 +1,5 @@
+// Package workloads provides a programmatic API for interacting with New Relic
```

Comment:
Yum, package docs.

Reply:
Oh I see there is also a `doc.go` above.  Are both points of documentation requried?

Reply:
No, just one will do... I'll clean this up in a future PR